lib/vquic/quiche.c

```diff
@@ -410,6 +412,9 @@ static CURLcode flush_egress(struct connectdata *conn, curl_socket_t sockfd,
     socksent = send(sockfd, out, sent, 0);
     if(socksent < 0) {
       if(errno == EAGAIN || errno == EWOULDBLOCK || errno == EINTR) {
+        FD_ZERO(&fds_write);
+        FD_SET(sockfd, &fds_write);
+        Curl_select(sockfd + 1, NULL, &fds_write, NULL, 0);
```

Comment:
This can't be right. If if can't send more now, it should do the non-blocking behavior and return EAGAIN all the way back and get called again later.